### PR TITLE
Back to groovy all

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -214,77 +202,8 @@ under the License.
...
-    </dependency>
-    <dependency>
-      <groupId>${groovy-groupId}</groupId>
-      <artifactId>groovy-testng</artifactId>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>${groovy-groupId}</groupId>
-      <artifactId>groovy-xml</artifactId>
+      <version>3.0.8</version>
```
Comment (pzygielo) [inline pom.xml:205]:
```suggestion
      <version>${groovy-version}</version>
```
?

**Comment by slawekjaranowski** [comment]
I'm not sure if we really need always all groovy dependencies. maybe better will be prepare example how to add more dependencies to plugin executions where is really need. I will check it in next week ...

**Comment by pzygielo** [comment]
> I'm not sure if we really need always all groovy dependencies. maybe better will be prepare example how to add more dependencies to plugin executions where is really need. I will check it in next week ...

Consult https://issues.apache.org/jira/browse/MINVOKER-138 which done exactly the opposite 10 years ago.

File: pom.xml (hunk 1 above)
Comment (olamy) [reply pom.xml]: arff good catch

**Comment by olamy** [comment]
@slawekjaranowski I don't understand what problem do you want to solve? I think it's fine to have full dependencies and easy life. removing that will just break a lot of users!!

**Pull request merged**